plugins/hypervisors/hyperv/src/org/apache/cloudstack/storage/motion/HypervStorageMotionStrategy.java

```diff
@@ -136,7 +137,7 @@ private Answer migrateVmWithVolumes(VMInstanceVO vm, VirtualMachineTO to, Host s
             }
 
             return answer;
-        } catch (OperationTimedoutException e) {
+        } catch (OperationTimedoutException | OperationCancelledException e) {
```

Comment:
The AgentUnavailableException says the message as operation timed out even for a cancelled exception, should it be handled properly?